CHANGELOG.md

```diff
@@ -4,6 +4,58 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.8.0] - 2024-08-19
+
+This release completely refactors the directory structure of the repository for a more seamless and intuitive developer journey. It also adds support to deploy latest accelerated embedding and reranking models across the cloud, data center, and workstation using [Nvidia NeMo Retriever NIMs](https://docs.nvidia.com/nim/index.html#nemo-retriever).
+
+### Added
+- [End to end RAG examples](./RAG/examples/) enhancements
+  - [Single command deployment](./README.md#try-it-now) for all the examples using docker compose.
+  - All end to end RAG examples are now more encapsulated with documentation, code and deployment assets residing in dedicated example specific directory.
+  - Segregated examples into [basic and advanced RAG](./RAG/examples/) with dedicated README's.
+  - Added reranker model support to [multi-turn RAG example](./RAG/examples/advanced_rag/multi_turn_rag/)
+  - Removed python dev packages from containers to enhance security
+  - Updated to latest version of [langchain-nvidia-ai-endpoints](https://python.langchain.com/v0.2/docs/integrations/providers/nvidia/)
+- [Speech support using rag-playground]((./docs/riva-asr-tts.md))
+  - Added support to access [RIVA speech models from NVIDIA API Catalog](https://build.nvidia.com/explore/speech).
+  - Speech support in UI is turned off as default.
```

Comment:
Can we please rephrase "tuned off by default" to "opt-in"? Thanks

Reply:
Changed!